web/styles/app_components.css

```diff
@@ -448,6 +448,22 @@ div.overlay {
     }
 }
 
+.bot-avatar {
+    position: relative;
+    overflow: hidden;
+
+    &::after {
+        content: " ";
```

Comment:
Some inline comments here on what you're doing and why to get the black-triangle effect will be a help to future contributors.